\documentclass[11pt]{article}
\usepackage[T1]{fontenc}
\usepackage{lmodern}

\usepackage[letterpaper,margin=1in]{geometry}

\usepackage{amsmath}
\usepackage{booktabs}
\usepackage{calc}
\usepackage{caption}
\usepackage[flushmargin,hang]{footmisc}
\usepackage{float}
\usepackage{microtype}
\usepackage{newverbs}
\usepackage{tablefootnote}
\usepackage{tabularx}
\usepackage{todonotes}
\usepackage[hyperfootnotes=false]{hyperref} % doesn't work in tabulars as currently set
\usepackage[nohyperlinks]{acronym}
\usepackage{footnotehyper}
\usepackage[strict]{changepage}
\usepackage[binary-units=true]{siunitx}
\usepackage{enumitem}
\usepackage{stackengine}


\input{bedRModv2.ver}

\hypersetup{colorlinks=true,
  linkcolor=blue,
  filecolor=magenta,
  urlcolor=blue,
  pdfinfo={githash=\commitdesc}}

\definecolor{cverbbg}{gray}{0.93}

\title{The \acf{bedRMod} format}
\author{Transregio 319 RMaP}
\date{\headdate}

\setlength{\emergencystretch}{\hsize}
\setlength{\footnotemargin}{1em}

\floatplacement{table}{htbp}
\setcounter{topnumber}{2}
\setcounter{bottomnumber}{2}
\setcounter{totalnumber}{4}
\setcounter{dbltopnumber}{2}
\renewcommand{\dbltopfraction}{0.9}
\renewcommand{\textfraction}{0.07}
\renewcommand{\floatpagefraction}{0.7}

\interfootnotelinepenalty=1000000
\makesavenoteenv{tabularx}

\newcolumntype{L}{>{\raggedright\arraybackslash}X}

\providecommand*{\Ac}[1]{\ac{#1}} % work around outdated acronym.sty packages
\newcommand*{\acrodefused}[2]{\acrodef{#1}{#2}\acused{#1}}

\frenchspacing

% eliminate passive voice warnings
% chktex-file -3

\begin{document}

\maketitle

\begin{small}
\noindent
The master version of this document can be found at \url{https://github.com/dieterich-lab/euf-specs}.
This printing is version~\commitdesc\ from that repository, last modified on the date shown above.
\end{small}

\acused{ASCII}

\section{Specification}

\Ac{bedRMod} formalizes the ENCODE bedMethyl format\footnote{Description of bedMethyl file, \url{https://www.encodeproject.org/data-standards/wgbs}} for \emph{RNA modification}s. \Ac{bedRMod} is a tab-delimited~\textbf{file} format, compatible with the \acf{BED} format\footnote{SAM/BAM and related specifications, \url{http://samtools.github.io/hts-specs}}. Metadata are in~\textbf{header line}s, which describe metainformation about the source of the data. Data are in~\textbf{data line}s, which describe \emph{RNA modification}s, or sites of putative \emph{RNA modification}s, by physical start and end position on a linear chromosome. The metadata must be consistent for all~\textbf{data line}s. The~\textbf{file} extension for the \ac{bedRMod} format is~\texttt{.bedrmod},~\texttt{.bedmethyl}, or~\texttt{.bed}.

\subsection{Scope}

This specification is a variation of the \ac{BED} description for~\textbf{data line}s. The content of this document is directly inspired from the 
official \ac{BED} specifications. Only the most important or less obvious concepts are reiterated in this document. For general information, refer to the official \ac{BED} specifications. Potential interoperability issues with the \ac{BED} format are also described in this document.

\subsection{Typographic conventions}

This document uses the official \ac{BED} typographic conventions~(\autoref{tab:typographic-conventions}).

\begin{savenotes}
  \begin{table}
    \begin{tabularx}{\textwidth}{r L L}
      \toprule
      Style & Meaning & Examples \\
      \midrule
      Bold & Terms defined in~\autoref{sec:terms} and~\ref{sec:lines} & \textbf{file}{\quad}\textbf{line} \\
      Sans serif & Names of~\textbf{field}s & \textsf{chrom}{\quad}\textsf{chromStart}{\quad}\textsf{chromEnd} \\
      Fixed-width & Literals or \ac{regex}es\footnote{POSIX/IEEE~1003.1--2017 Extended Regular Expressions, for the ``C'' locale.
                    \emph{IEEE Standard for Information Technology---Portable Operating System Interface~(POSIX) Base Specifications}, IEEE~1003.1--2017, 2017} & \texttt{.bedrmod}{\quad}\texttt{grep}{\quad}\texttt{[[:alnum:]]+} \\
      \bottomrule
    \end{tabularx}
    \caption{\textbf{Typographic conventions.}}\label{tab:typographic-conventions}
  \end{table}
\end{savenotes}

\subsection{Terminology and concepts}\label{sec:terms}
\begin{description}
\item[0-based, half-open coordinate system:]
  A coordinate system where the first base starts at position~0, and the start of the interval is included but the end is not.
  For example, for a sequence of bases~\texttt{ACTGCG}, the bases given by the interval~[2,~4) are~\texttt{TG}. % chktex 9

\item[\acs{bedRMod} field:]
  One of the 11~standard~\textbf{field}s defined in this specification.
  All~\textbf{\acs{bedRMod} field}s are mandatory.

\item[comment line:]
  A~\textbf{line} that starts with~\texttt{\#} with no horizontal whitespace beforehand. \textbf{Comment line}s at the start of 
  the~\textbf{file} are~\textbf{header line}s defined in this specification.

\item[custom field:]
  A~\textbf{field} defined by the~\textbf{file}~creator.
  \textbf{Custom field}s occur in each~\textbf{line} after any~\textbf{\acs{bedRMod} field}s.

\item[data line:]
  A~\textbf{line} that contains~\textbf{feature}~data.

\item[feature:]
  A linear region of a chromosome reporting a \emph{RNA modification}, or a site of putative \emph{RNA modification}, supported by quantitative evidence, typically at single-base resolution, but can include a context.

\item[field:]
  Data stored as non-tab text.
  All~\textbf{field}s are 7-bit US \ac{ASCII} printable characters\footnote{Characters in the range~\texttt{{\textbackslash}x20} to~\texttt{{\textbackslash}x7e}, therefore not including any control characters}.

\item[field separator:]
  One or more horizontal whitespace characters (space or tab).
  The~\textbf{field separator} must match the \ac{regex}~\texttt{[ {\textbackslash}t]+}.
  This specification strongly recommends using tab as~\textbf{field separator} throughout the~\textbf{file}.

\item[file:]
  Sequence of one or more~\textbf{data line}s with a~\textbf{header}.

\item[header:]
  Mandatory~\textbf{header line}s, followed by optional~\textbf{comment line}s, at the start of the~\textbf{file}, \textit{cf.}~\autoref{sec:header}.

\item[header field:]
  A mandatory~\emph{key=value} pair describing one of the~\textbf{header line}s. 
  
\item[header line:]
  A~\textbf{line} that starts with~\texttt{\#} with no horizontal whitespace beforehand, immediately followed by a~\textbf{header field}.
  
\item[line:]
  String terminated by a~\textbf{line separator}, in one of the following classes.
  Either a~\textbf{data line} or a~\textbf{comment line}, \textit{cf.}~\autoref{sec:lines}.

\item[line separator:]
  Either carriage return~(\texttt{{\textbackslash}r}, equivalent to~\texttt{{\textbackslash}x0d}), newline~(\texttt{{\textbackslash}n}, equivalent to~\texttt{{\textbackslash}x0a}), or carriage return followed by newline~(\texttt{{\textbackslash}r{\textbackslash}n}, equivalent to~\texttt{{\textbackslash}x0d{\textbackslash}x0a}).
  The same~\textbf{line separator} must be used throughout the~\textbf{file}.
\end{description}

\subsection{Lines}\label{sec:lines}

\subsubsection{Data lines}

\textbf{Data line}s contain~\textbf{feature}~data.
A~\textbf{data line} is composed of~\textbf{field}s separated by~\textbf{field separator}s.

\subsubsection{Comment lines}

\textbf{Comment line}s provide no~\textbf{feature} data. They start with~\texttt{\#} with no horizontal whitespace beforehand.
\textbf{Comment line}s at the beginning of the~\textbf{file} are treated as~\textbf{header line}s, and must conform to~\textbf{header} specifications. A~\texttt{\#} appearing anywhere else in a~\textbf{data line} is treated as~\textbf{feature} data, not a comment.


\subsection{Header specification}\label{sec:header}

The~\textbf{header} contains metainformation about the source of the data. Each~\textbf{header line} starts with a~\texttt{\#} and contains a
mandatory~\textbf{header field} in the form of a~\emph{key=value} pair~(\autoref{tab:header}). All~\textbf{header field}s are mandatory. The first seven~\textbf{header field}s must be assigned a value, and the value must follow a controlled vocabulary, see~\autoref{sec:recommend-header} and~\ref{sec:recommend-fields} for examples and recommendations. Additional~\textbf{line}s starting with~\texttt{\#} are treated as~\textbf{comment line}s.

A \ac{bedRMod}~\textbf{header} describes information for one organism, one assembly and annotation, and one modification (RNA) type, hence a \ac{bedRMod}~\textbf{file} contains~\textbf{data lines} for one organism, one assembly and annotation, and one modification (RNA) type, \textit{e.g.} a \ac{bedRMod}~\textbf{file} contains~\textbf{data lines} for~\texttt{m6A} and~\texttt{m5C} in human mRNA, using~\texttt{GRCh38} and ~\texttt{Ensembl 110}.

\begin{savenotes}
  \begin{table}
    \begin{tabularx}{\textwidth}{X p{.5\textwidth} p{.15\textwidth}}
      \toprule
      Header field key & Brief description & Value required \\
      \midrule
      \textsf{fileformat} & Fileformat and version & Yes \\
      \textsf{organism} & NCBI Taxonomic identifier\footnote{NCBI Taxonomy: a comprehensive update on curation, resources and tools, \url{https://doi.org/10.1093/database/baaa062}} & Yes \\
      \textsf{modification\textunderscore type} & A valid RNA type & Yes \\
      \textsf{modification\textunderscore names} & name:short\textunderscore name:primary\textunderscore base & Yes \\
      \textsf{assembly} & Genome assembly & Yes \\
      \textsf{annotation\textunderscore source} & Annotation source & Yes \\
      \textsf{annotation\textunderscore version} & Annotation version & Yes \\
      \textsf{sequencing\textunderscore platform} & Sequencing platform & No \\
      \textsf{basecalling} & Basecalling model information where relevant & No \\
      \textsf{bioinformatics\textunderscore workflow} & Link to bioinformatics workflow; program name, version, and/or call; information relevant to score, coverage, or frequency calculation; \textit{etc.} & No \\
      \textsf{experiment} & Information about or link to experimental protocol and design & No \\
      \textsf{external\textunderscore source} & Databank:ID of data & No \\
      \bottomrule
    \end{tabularx}
    \caption{\textbf{Header Fields.}}\label{tab:header}
  \end{table}
\end{savenotes}
 

\subsection{Data specification}\label{sec:data}

Each~\textbf{data line} contains 11~\textbf{\acs{bedRMod} field}s delimited by a~\textbf{field separator} (tab).
All~\textbf{fields} are mandatory~(\autoref{tab:fields}). Missing data is not allowed.
Additional optional~\textbf{field}s can be added, following the first 11~\textbf{field}s, according to the \acs{BED} specifications, but 
it is not recommended to use \acs{bedRMod} with exactly 12~\textbf{field}s, \textit{cf.}~\autoref{sec:custom_fields}.

\begin{savenotes}
  \begin{table}[H]
    \begin{adjustwidth}{-0.5in}{-0.5in}
      \begin{tabularx}{\linewidth}{r l l l L}
        \toprule
        Col & \acs{bedRMod} field & Type & Regex or range & Brief description \\
        \midrule
        1
        & \textsf{chrom}
        & String
        & \texttt{[[:alnum:]\_]\{1,255\}}\footnote{\texttt{[[:alnum:]\_]} is equivalent to the \ac{regex}~\texttt{[A-Za-z0-9\_]}. % chktex 8
        It is also equivalent to the Perl extension~\texttt{[[:word:]]}}
        & Chromosome name \\
        2 & \textsf{chromStart} & Int & $[0, 2^{64}-1]$ & \textbf{Feature} start position \\
        3 & \textsf{chromEnd} & Int & $[0, 2^{64} -1]$ & \textbf{Feature} end position \\
        4 
        & \textsf{name} 
        & String 
        & \texttt{[{\textbackslash}x20-{\textbackslash}x7e]\{1,255\}} 
        & \textbf{Feature} name and additional attributes \\
        5 & \textsf{score} & String & \texttt{[{\textbackslash}x20-{\textbackslash}x7e]\{1,255\}} & \textbf{Feature} confidence \\
        6 & \textsf{strand} & String & \texttt{[-+.]} & \textbf{Feature} strand \\
        7 & \textsf{thickStart} & Int & $[0, 2^{64}-1]$ & Thick start position, typically same as~\textsf{chromStart} \\
        8 & \textsf{thickEnd} & Int & $[0, 2^{64}-1]$ & Thick end position, typically same as~\textsf{chromEnd} \\
        9 & \textsf{itemRgb} & Int,Int,Int & \texttt{(}$[0, 255], [0,255], [0,255]$\texttt{) | 0} & Display color \\ % chktex 9
        10 & \textsf{coverage} & Int &  $(0, 2^{64}-1]$ & \textbf{Feature} coverage \\
        11 & \textsf{frequency} & Float & $[0, 100]$ & \textbf{Feature} frequency, \textit{i.e.} percentage of modification \\
        \bottomrule
      \end{tabularx}
    \end{adjustwidth}
    \caption{\textbf{\acs{bedRMod} Fields.}}\label{tab:fields}
  \end{table}
\end{savenotes}

In a \ac{bedRMod}~\textbf{file}, each~\textbf{data line} must have the same number of~\textbf{field}s.
The positions in~\textbf{\acs{bedRMod} field}s are all described in the~\textbf{0-based, half-open coordinate system}, exactly as 
described in the official \ac{BED} specifications.

\subsection{Coordinates}
Refer to the official \ac{BED} specifications.

\subsection{Simple attributes}
\begin{enumerate}
\item \textsf{name}: String that describes the~\textbf{feature}, \textit{i.e.} the modification. \textsf{name} must describe 
the modification using the \emph{short name} of the MODOMICS nomenclature\footnote{MODOMICS, \url{https://www.genesilico.pl/modomics/modifications}}, or the base modification code described in the SAMtags\footnote{SAM tags, \url{https://samtools.github.io/hts-specs}}, or a numeric ChEBI code\footnote{Chemical Entities of Biological Interest, \url{https://www.ebi.ac.uk/chebi}}. The MODOMICS \emph{short name} corresponding to~\textsf{name} is always described in the~\textbf{header field}~\textsf{modification\textunderscore names}. Additional name attributes are allowed, and must be comma-separated \textit{e.g.}~\texttt{a,DRACH,2}.

\item \textsf{score}: String representation of the confidence in calling this modification. Any measure of confidence is valid, but a \ac{bedRMod}~\textbf{file} with non-integer-like~\textsf{score} values outside the range $[0, 1000]$ may fail from being correctly displayed in a visual representation\footnote{\textit{cf.} bedtools definition of score, \url{https://bedtools.readthedocs.io/en/latest/content/general-usage.html?highlight=bed\%20format}}.

\item \textsf{coverage}: Integer between~0 and the maximum size of an unsigned 64-bit integer, excluding~0, representing the number of reads with a base aligned to this reference position for which this~\textbf{feature} is a modification. The primary or canonical base must be inferred by the modification, \textit{e.g.} for ~\texttt{m6A}, this is the number of reads with an~\texttt{A} aligned to this position.

\item \textsf{frequency}: Float\footnote{Decimal string representation of 64-bit floating point number, IEEE Standard for Binary Floating-Point Arithmetic. IEEE 754â€“1985, 1985.} between~0 and~100, including~0, representing the modification frequency, or stoichiometry. This can be the percentage of modified reads, or the ratio of the number of calls passing filters that were classified as a residue with the base modification reported for this~\textbf{feature} to the \emph{valid coverage}, multiplied by~100. See~\autoref{sec:fields} for an explanation of \emph{valid coverage}. A~\textsf{frequency} of~0 means that there is evidence that a given site is not modified, \textit{i.e.} the primary or canonical base is reported with a confidence quantified by a~\textsf{score}~$>$0.
\end{enumerate}

\subsection{Display attributes}
\begin{enumerate}
  \setcounter{enumi}{4}

\item \textsf{thickStart}: Included for compatibility, typically same as~\textsf{chromStart}.

\item \textsf{thickEnd}: Included for compatibility, typically same as~\textsf{chromEnd}.

\item \textsf{itemRgb}: Included for compatibility, typically~\texttt{0,0,0}.

\end{enumerate}

\subsection{Custom fields}\label{sec:custom_fields}

\textbf{Custom field}s defined by the~\textbf{file} creator may contain any printable 7-bit US \ac{ASCII} character (which includes spaces, but excludes tabs, newlines, and other control characters), as defined by the \ac{BED} format definitions.

A \acs{bedRMod}~\textbf{file} with exactly 12~\textbf{field}s, \textit{i.e.} containing one additional optional~\textbf{field}, may be implicitely 
assumed to be a~\textbf{BED12}~\textbf{file} by certain software and genome browsers, which can result in unexpected behaviour! 

\section{Examples}

\subsection[title]{Example bedRMod file from the \acs{bedRMod} and related specifications\footnote{\url{https://github.com/dieterich-lab/euf-specs/examples/bedrmod/example.bedrmod}}}\label{sec:example-bedrmod}

\begin{verbatim}
#fileformat=bedRModv2
#organism=9606
#modification_type=RNA
#modification_names=20607:m5C:C,21891:m6A:A
#assembly=GRCh38
#annotation_source=Ensembl
#annotation_version=93
#sequencing_platform=Illumina NovaSeq 6000
#basecalling=
#bioinformatics_workflow=workflow:https://github.com/XXX
#experiment=https://doi.org/10.XXX
#external_source=SRA:PRJNAXXXXXX,GEO:GSEXXXXXX
#chrom chromStart chromEnd name score strand thickStart thickEnd itemRgb coverage frequency
1 1391918 1391919 20607 20 - 1391918 1391919 0,0,0 42 42.56
2 8878712 8878713 20607 150 - 8878712 8878713 0,0,0 318 44.23
3 11980442 11980443 21891 78 + 11980442 11980443 0,0,0 111 56.20
4 17054111 17054112 20607 10 - 17054111 17054112 0,0,0 40 34.03
\end{verbatim}

\section{Recommended practice for the \acs{bedRMod} format}

\subsection{\acs{bedRMod} extension}\label{sec:recommend-ext}
The~\textbf{file} extension is~\texttt{.bedrmod},~\texttt{.bedmethyl}, or~\texttt{.bed}. Since the \ac{BED} format prohibits~\textbf{BED11}, there should be little confusion in general, but when~\textbf{custom field}s are defined, it is recommended to use the~\texttt{.bedrmod} or~\texttt{.bedmethyl} extension.

\subsection{Mandatory \acs{bedRMod} header fields}\label{sec:recommend-header}
These~\textbf{header field}s are not free text, and must conform to a controlled vocabulary.
\begin{itemize}
\item \textsf{fileformat}: A valid version of this specification, including the format name, \textit{e.g.}~\texttt{bedRModv2}. 

\item \textsf{modification\textunderscore type}: A valid RNA type\footnote{For example RNA or mRNA, tRNA, or rRNA. A next version of this specification should prescribe a controlled vocabulary by providing a reference to an established RNA ontology.}.

\item \textsf{modification\textunderscore names}: A comma-separated dictionary mapping in the form name:short\textunderscore name:primary\textunderscore base, where name corresponds to the~\textsf{name}~\textbf{field} of a~\textbf{data line}, short\textunderscore name to the \emph{short name} of the MODOMICS nomenclature, and primary\textunderscore base is the canonical or primary sequence base, \textit{e.g.}~\texttt{21891:m6A:A}. A value is also required when using MODOMICS \emph{short names}, \textit{e.g.}~\texttt{m6A:m6A:A}. All modifications present in the~\textbf{file} must be included, in a comma-separated list of items.

\item \textsf{organism}: A valid NCBI Taxonomic identifier, \textit{e.g.}~\texttt{9606}.

\item \textsf{assembly}: The name of a valid assembly, \textit{e.g.} using the Ensembl terminology,~\texttt{GRCh38}. 

\item \textsf{annotation\textunderscore source}: The name of a valid annotation, \textit{e.g.}~\texttt{Ensembl}. 

\item \textsf{annotation\textunderscore version}: A valid version for the annotation source, \textit{e.g.}~\texttt{110}. 
\end{itemize}

\subsection{\acs{bedRMod} header fields}\label{sec:recommend-fields}
These~\textbf{header field}s can be left without a value, but the key must always be present. The value is free text, although it is strongly recommended
to reference established \textsf{sequencing\textunderscore platform}s, \textsf{basecalling} models, \textsf{bioinformatics\textunderscore workflow}s, or \textsf{external\textunderscore source}s using an exact terminology and/or recognized identifiers.  
\begin{itemize}
\item \textsf{sequencing\textunderscore platform}: Typically, the name of the sequencing instrument or device, including key specifications if relevant, \textit{e.g.}~\texttt{ONT MinION}.

\item \textsf{basecalling}: Basecalling model such as name of versioned model, reference to published model, and/or additional details on training, \textit{e.g.}~\texttt{dna\textunderscore r9.4.1\textunderscore e8\textunderscore sup$@$v3.6}.

\item \textsf{bioinformatics\textunderscore workflow}: Program name, version, and/or call used to generate the~\textbf{file}, or link to open source bioinformatics workflow, including version and/or any additional details to facilitate data lineage. The information should be sufficient to reproduce the content of the~\textbf{file}.

\item \textsf{experiment}: Supplementary information about experimental protocol, design, or the content of the~\textbf{file} such as conditions used, number of replicates, \textit{etc.}, or link to an openAIRE repository.

\item \textsf{external\textunderscore source}: A comma-separated list of sources of the form Databank:ID, \textit{e.g.}~\texttt{GEO:GSEXXXXXX, Zenodo:10.XXX/zenodo.XXXXXXXX}. Free text is allowed, but this specification recommends using the format described here. Together with~\textsf{bioinformatics\textunderscore workflow}, this should allow to reproduce the content of the~\textbf{file}.
\end{itemize}


\subsection{\acs{bedRMod} fields}\label{sec:fields}
\begin{itemize}
\item \textsf{chrom}: The name of each chromosome should match the names from a reference genome assembly, as given in the~\textbf{header}. For example, if~\texttt{\#assembly=GRCh38}, then chromosomes should be named~\texttt{1} to~\texttt{22}, \texttt{X}, \texttt{Y}, and~\texttt{MT}, consistently through the~\textbf{file}.

\item \textsf{name}: The MODOMICS \emph{short names} or the ChEBI codes should be used in preference to the base modification codes described in the SAMtags.

\item \textsf{score}: The \emph{valid coverage} should be used as a measure of confidence. This can be \textit{e.g.} the number of calls passing filters (classified as modified and unmodified) at the reported modification position\footnote{\textit{cf.} \url{https://nanoporetech.github.io/modkit}}, or the number or reads remaining after filtering and used to infer the modificaton status.
\end{itemize}

\subsection{Whitespace}\label{sec:whitespace}
We recommend that only a single tab~(\texttt{{\textbackslash}t}) be used as~\textbf{field separator}, \textit{cf.} offical \ac{BED} specifications.

\section{Information supplied out-of-band}

A \ac{bedRMod} \textbf{file} contains 11 required~\textbf{field}s, any additional~\textbf{field}s may require information that must be supplied out-of-band. A common practice is to include a~\textbf{comment line} after the~\textbf{header} to describe the~\textbf{field}s used in the~\textbf{file}, \textit{cf.}~\autoref{sec:example-bedrmod}.

The semantics of~\textbf{field}s such as~\textsf{score},~\textsf{coverage}, and~\textsf{frequency} can be included in the~\textsf{header} using the \textbf{header field}~\texttt{bioinformatics\textunderscore workflow}.

\section{Acronyms}

% using the optional argument to acronym to set the label width causes it to use the list environment instead of description, which means we can't set nosep easily
\setlist[description]{labelwidth=\widthof{\textbf{\acs{bedRMod}}},nosep}
\begin{acronym}
  \acro{ASCII}{American Standard Code for Information Interchange}
  \acro{BED}{Browser Extensible Data}
  \acro{bedRMod}{Browser Extensible Data for RNA modification}
  \acro{ChEBI}{Chemical Entities of Biological Interest}
  \acro{NCBI}{National Center for Biotechnology Information}
  \acro{regex}{regular expression}
\end{acronym}

\section{Acknowledgments}

We thank the \acf{bedRMod} format specification working group and the Modkit developers.

\end{document}

% chktex-file 17

%%% Local Variables:
%%% mode: latex
%%% TeX-master: t
%%% End:
